src/pages/contact/contact.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import './styles.css'
+
+export function Contact() {
+
+    return(<div className="contact">
+        <h1>Contact</h1>
+    </div>
+    
```

Comment:
Remove whitespace, please use PRETTIER plugin for formatting

Reply:
the page has been removed